app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewCard.kt

```diff
@@ -7,49 +7,51 @@ import androidx.compose.foundation.layout.wrapContentSize
 import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material.Card
 import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.draw.clip
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 
-data class CrewCardCardViewState(
+data class CrewCardViewState(
     val name: String,
     val role: String,
 )
 
 @Composable
 fun CrewCard(
-    crewCardCardViewState: CrewCardCardViewState,
+    crewCardCardViewState: CrewCardViewState,
     modifier: Modifier = Modifier,
 ) {
-    Surface(
-        color = MaterialTheme.colors.background,
-        modifier = modifier.padding(10.dp)) {
-        Card(shape = RoundedCornerShape(4.dp)) {
-            Column(
+    Card(
+        shape = RoundedCornerShape(4.dp),
+        modifier = modifier.padding(10.dp)
+    ) {
+        Column(modifier = Modifier.wrapContentSize())
+        {
```

Comment:
formatting :D check `Card` component above - `{` always goes after the closing parentheses